wf/ja-jp/content/topics/workflows/execute/flow-api-endpoint.htm

```diff
@@ -224,43 +224,40 @@
                                             <div class="dropdown" data-test="menuBarCategory" id="documentationMenu" data-mc-conditions="MultiProdPublish.DocMenu">
                                                 <button class="dropbtn">ドキュメント <i class="fa fa-caret-down"></i></button>
                                                 <div class="dropdown-content" data-test="menuBarItems">
-                                                    <div><a href="https://help.okta.com/ja-jp/Content/index.htm" data-test="menuBarLink">製品ドキュメント</a>
+                                                    <div><a href="https://help.okta.com/okta_help.htm?type=oie&amp;id=oie-index" data-test="menuBarLink">Okta Identity Engine</a>
```

Comment:
@pengyao-okta This topic (and many others) does not have the correct label

Reply:
The menu dropdowns should be consistent, "Identity. Engine" etc., no Okta

Reply:
How do they return different strings for the same input in one translation job? 😞
Also if we add something like `no-translate` attribute to DOM element would that help XTM be more consistent?